Updated emac greentea tests

mikaleppanen opened the pull request:
### Description

- Improved markdown
- Set tests to fail if CTP echo server is not present

### Pull request type

    [X] Fix
    [ ] Refactor
    [ ] New target
    [ ] Feature
    [ ] Breaking change

Commit:
Updated emac greentea tests
- Improved markdown
- Set tests to fail if CTP echo server is not present

mikaleppanen commented:
Disabled also "printf("cable connected\r\n\r\n");" traces could result problems with greentea tracing/test case execution.

maclobdell commented:
I just tested this on two K64F.   Everything passing now.

```
+--------------+---------------+--------------------+---------------------------+--------+--------+--------+--------------------+
| target       | platform_name | test suite         | test case                 | passed | failed | result | elapsed_time (sec) |
+--------------+---------------+--------------------+---------------------------+--------+--------+--------+--------------------+
| K64F-GCC_ARM | K64F          | tests-network-emac | EMAC broadcast            | 1      | 0      | OK     | 10.29              |
| K64F-GCC_ARM | K64F          | tests-network-emac | EMAC initialize           | 1      | 0      | OK     | 1.89               |
| K64F-GCC_ARM | K64F          | tests-network-emac | EMAC memory               | 1      | 0      | OK     | 293.97             |
| K64F-GCC_ARM | K64F          | tests-network-emac | EMAC multicast filter     | 1      | 0      | OK     | 62.1               |
| K64F-GCC_ARM | K64F          | tests-network-emac | EMAC unicast              | 1      | 0      | OK     | 4.28               |
| K64F-GCC_ARM | K64F          | tests-network-emac | EMAC unicast burst        | 1      | 0      | OK     | 53.9               |
| K64F-GCC_ARM | K64F          | tests-network-emac | EMAC unicast frame length | 1      | 0      | OK     | 5.9                |
+--------------+---------------+--------------------+---------------------------+--------+--------+--------+--------------------+
```

Commit:
Copy edit README.md

Copy edit changes to README.md, mostly for active voice, consistent tense across docs and style.

0xc0170 commented:
/morph build
